yarn.lock

```diff
@@ -40,6 +40,67 @@
     signedsource "^1.0.0"
     yargs "^15.3.1"
 
+"@aws-amplify/amplify-appsync-simulator@2.9.1":
```

Comment:
These deps were pulled in from running `yarn setup-dev`, could you rerun w/ just a `yarn` so we're not adding back in the CLI packages to the lockfile?